bootstrap/README.md

```diff
@@ -21,47 +21,57 @@ and based on the results chooses good values for various Kubeflow parameters.
 
 **Alpha stage(as of today) Requires** run ```make build``` to build docker image locally since there's no public release yet.
 
-Interactive use
+**Enter interactive-use container**:
 
 ```
 TAG=latest
 APP_DIR_HOST=<Directory on host machine for the ksonnet apps>
```

Comment:
`APP_DIR_HOST=$HOME/ksonnetApps # Directory on host machine for the ksonnet apps`
It will be good to provide a default location so that user can just copy-paste